bindings/python/generate_bindings.py

```diff
@@ -1,4 +1,82 @@
 import json
+import argparse
+import sys
+import os
+
+
+class kind:
+
+    KIND = None
+    NAME = None
+    MEMBERS = None
+    DEPTH = None
+    state_stack = []
+    kind_functions = {
+        "TRANSLATION_UNIT": [],
+        "NAMESPACE": [handle_NAMESPACE_0, handle_NAMESPACE_1],
+        "STRUCT_DECL": [handle_STRUCT_DECL_0, handle_STRUCT_DECL_1],
+        "CXX_BASE_SPECIFIER": [],
+        "CXX_METHOD": [handle_CXX_METHOD],
+        "CONSTRUCTOR": handle_CONSTRUCTOR,
+        "VAR_DECL": handle_VAR_DECL,
+        "PARM_DECL": handle_PARM_DECL,
+        "TYPE_REF": handle_TYPE_REF,
+        "CALL_EXPR": handle_CALL_EXPR,
+    }
+    linelist = []
+
+    def handle_kind(self, item):
+        KIND = item["kind"]
+        NAME = item["name"]
+        MEMBERS = item["members"]
+        DEPTH = item["depth"]
+
+        self.state_stack.append({"kind": KIND, "name": NAME, "depth": DEPTH})
+
+        if self.kind_functions[KIND][0]:
+            self.kind_functions[KIND][0](item)
+
+        for item in MEMBERS:
+            self.handle_kind(item)
+
+        if self.kind_functions[KIND][1]:
+            self.kind_functions[KIND][1](item)
+
+        self.state_stack.pop()
+
+    def handle_NAMESPACE_0(self, item):
+        self.linelist.append(f"namespace {self.NAME}" + "{")
+
+    def handle_NAMESPACE_1(self, item):
+        self.linelist.append("}")
+
+    def handle_STRUCT_DECL_0(self, item):
+        MEMBERS = item["members"]
+        for item in MEMBERS:
+            if item["kind"]=="CXX_BASE_SPECIFIER":
+                CXX_BASE_SPECIFIER = item["name"]
+                self.linelist.append(f'py::class_<{self.NAME, CXX_BASE_SPECIFIER}>(m, "{self.NAME}")')
+                return
```

Comment:
You're handling just 1 base class (and only the base class in this function)? Please note so with a comment